apps/ashley/templates/services/app/job_01_collectstatic.yml.j2

```diff
@@ -22,9 +22,10 @@ spec:
         deployment_stamp: "{{ deployment_stamp }}"
         job_stamp: "{{ job_stamp }}"
     spec:
-{% if ashley_image_pull_secret_name is defined and ashley_image_pull_secret_name is not none %}
+{% set image_pull_secret_name = ashley_image_pull_secret_name or default_image_pull_secret_name %}
++{% if image_pull_secret_name is defined and image_pull_secret_name is not none %}
```

Comment:
```suggestion
{% if image_pull_secret_name is defined and image_pull_secret_name is not none %}
```

Reply:
This error is repeated multiple times.

Reply:
yes it was a regex search/replace :pray: